client/src/App.js

```diff
@@ -1,12 +1,56 @@
 import React, { useEffect } from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import store from './app/store';
 import { getUserData } from './features/User';
+import Nav from './Components';
 
 function App() {
   useEffect(() => {
     store.dispatch(getUserData());
   }, []);
-  return <div className="App">Hello World</div>;
+  return (
+    <Router>
+      <Nav />
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <h1>Home</h1>
+            }
+        />
+        <Route
+          path="/about-us"
+          element={
+            <h1>about-us</h1>
+            }
```

Comment:
We don't have `about us` page or contact, you can check the `Frontend Routes` issue